src/renderer.ts

```diff
@@ -82,7 +86,6 @@ export class Renderer {
       'setShadowBlur',
       'setLineWidth',
       'setLineCap',
-      'setLineDash',
```

Comment:
`setLineDash` already exists on `CanvasRenderingContext2D`, so this code does nothing. Removing it doesn't affect the test output.

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash

Reply:
Thanks, sounds good.